universal-editor-product-picker/src/universal-editor-ui-1/web-src/src/components/ExtensionRegistration.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * <license header>
+ */
+
+import { Text } from "@adobe/react-spectrum";
+import { register } from "@adobe/uix-guest";
+import { extensionId } from "./Constants";
+
+function ExtensionRegistration() {
+  const init = async () => {
+    await register({
+      id: extensionId,
+      methods: {
+        canvas: {
+          getRenderers() {
+            return [
+              {
+                extension: 'uix-product-picker',
+                dataType: 'text',
```

Comment:
Probably there should be a default datatype `product` or `category` that can be configured.